doc/nrf/known_issues.rst

```diff
@@ -865,7 +875,7 @@ NCSDK-7982: partition manager: Incorrect partition size linkage from name confli
 DFU and FOTA
 ============
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```

Comment:
This seems to have been fixed a while ago (oct 2020)

https://github.com/nrfconnect/sdk-nrf/pull/3197